sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1552,45 +1554,50 @@ private void updateCaseWithBulkData(
 		}
 	}
 
-	private CaseDataDto saveCase(@Valid CaseDataDto dto, boolean handleChanges, boolean checkChangeDate) throws ValidationRuntimeException {
-
+	public CaseDataDto saveCase(@Valid CaseDataDto dto, boolean handleChanges, boolean checkChangeDate) {
 		return saveCase(dto, handleChanges, checkChangeDate, true);
 	}
 
-	public CaseDataDto saveCase(@Valid CaseDataDto dto, boolean handleChanges, boolean checkChangeDate, boolean syncShares)
+	public CaseDataDto saveCase(@Valid CaseDataDto dto, boolean handleChanges, boolean checkChangeDate, boolean internal)
 		throws ValidationRuntimeException {
```

Comment:
I don't really like the paramter name `internal`, but I couldn't find a better name, If you know a better one I would change it.

It is used to skip edit allowed check and don't sync with other instances in case the update comes from another sormas instance.